packages/syft/src/syft/core/tensor/lazy_repeat_array.py

```
@@ -66,6 +68,14 @@ def __init__(self, data: np.ndarray, shape: Tuple[int, ...]) -> None:
                 if val < 0:
                     raise ValueError(f"Invalid shape: {shape}")
 
+        if self.data.size > 1:
+            if data_type == "min_val":
+                self.data = data.min()
+            elif data_type == "max_val":
+                self.data = data.max()
+            # else:
+            #     raise NotImplementedError("What on earth are you using this class for?")
```

We should raise an exception here :)